dashboard/app/assets/stylesheets/application.scss

```diff
@@ -230,12 +230,6 @@ img.video_thumbnail {
     }
   }
 
-  @media screen and (max-width: 1120px) {
-    .create_menu {
-      display: none !important;
-    }
-  }
-
```

Comment:
please revert the change to this file. Kelby's most recent change to this file adds this code back, and I don't think we should be letting any non-content changes through the DTS anyway.

Reply:
Thanks for catching that. Looks like I should also revert the change to `shared/css/user-menu.scss`?

Reply:
@sureshc correct, and thanks for noticing this @davidsbailey!